src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck start working!");
```

Comment:
The message 'Truck start working!' should be more informative and indicate that it is a Truck that is working. Consider rephrasing it to include the type of machine, for example: 'Truck is now working.'